azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/MessagingFactory.java

```diff
@@ -104,11 +104,18 @@ public void onReactorInit(Event e)
         Timer.register(this.getClientId());
 	}
 
-	String getHostName()
+	@Override
+	public String getHostName()
 	{
 		return this.hostName;
 	}
 	
+	@Override
+	public int getPort()
+	{
+		return ClientConstants.AMQPS_PORT;
```

Comment:
>AMQPS_PORT [](start = 25, length = 10)

What about websocket?

Reply:
Will tweak this when porting the changes to 2.0.

---
In reply to: [242724535](https://github.com/Azure/azure-service-bus-java/pull/313#discussion_r242724535) [](ancestors = 242724535)